subsys/bluetooth/host/hci_core.c

```diff
@@ -1252,7 +1252,7 @@ static void le_conn_complete_cancel(uint8_t err)
 		int busy_status = k_work_delayable_busy_get(&conn->deferred_work);
 
 		if (!(busy_status & (K_WORK_QUEUED | K_WORK_DELAYED))) {
-			/* Connection initiation timeout triggered. */
+			LOG_INF("Connection creation timeout triggered");
```

Comment:
Let's not use `LOG_INF()` for this, please. It should probably be `LOG_WRN()` or `LOG_DBG()`. `INF()` should generally be limited to very few select cases (I think there's just one or two of them right now in the entire Bluetooth subsystem).

Reply:
Again, I really disagree with the usage of log levels. But this is not the forum for that discussion, so @rugeGerritsen could you use `LOG_DBG`?

Reply:
Do we have some guidelines on what the different log levels should be used for?
Timing out establishing a connection can happen regularly in a noisy environment. To me personally `LOG_WRN` sounds like unlikely condition is happening, and something is going wrong which is not really the case.

Also, I would not use `LOG_DBG` here.
Scenario we have today:
1. User has developed an application with default logging configuration enabled
2. User gets connected callback with status 0x02
3. User does understand what is happening.

If we use `LOG_DBG` the user experience will not get improved. If the user is really interested in finding out what is going on, they can of course enable `CONFIG_BT_HCI_CORE_LOG_LEVEL_DBG`. But then they will likely drown in other logs. 

But if we have to choose between `LOG_DBG` and `LOG_WRN` I would choose `LOG_WRN`.

Reply:
I agree with the usage of `LOG_DBG`. This isn't an error or something unexpected, and the application gets the error code as well.

Reply:
> Do we have some guidelines on what the different log levels should be used for?

https://docs.zephyrproject.org/latest/services/logging/index.html#c.LOG_ERR

Reply:
> I agree with the usage of LOG_DBG. This isn't an error or something unexpected, and the application gets the error code as well.

If we decide to log this with `LOG_DBG`, the user experience will not get improved which is the point of this commit.

Reply:
I am OK with LOG_WRN as the description is

> It’s meant to register messages related to unusual situations that are not necessarily errors.

Which fits this scenario. 

> If we decide to log this with LOG_DBG, the user experience will not get improved which is the point of this commit.

Then you are also assuming that the user has enabled warning logs while not having enabled debug logs :) Since the log levels are configurable, we can't really assume what the user will see

Reply:
Yes. I have assumed the user only enables `CONFIG_LOG=y` which doesn't enable debug logs

Reply:
assumption is true. Zephyr is not chatty by default, unlike the nRF5 sdk